documentation/modules/auxiliary/admin/http/hikvision_unauth_pwd_reset_cve_2017_7921.md

```diff
@@ -0,0 +1,103 @@
+## Vulnerable Application
+
+Many Hikvision IP cameras contain a backdoor that allows unauthenticated impersonation of any
+configured user account. This is possible due to the firmware containing a hard coded magic string
+that bypasses all security on the camera and will provide full admin access. This allows a hacker to
```

Comment:
As a heads up we generally say an attacker and not a hacker. Hackers can be tinkers, creative people, or all sorts of things that don't fall under the definition of an attacker per say, so we would prefer to be more specific rather than using such a general term as "hacker" which may offend some people.

I'll go ahead and edit this up but this is similar to the idea behind using the terms denylist/allowlist over something like blacklist/whitelist, in that we want to try and be as inclusive and unoffensive in our language as possible when writing these documentations.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17033/commits/0fd7b51f7e7ce70adbbbab26c968d7a738fd40ff

Reply:
Makes sense... Attacker or bad actor is indeed a much better description.